rest/data/subscription.go

```diff
@@ -81,6 +82,39 @@ func (dc *DBSubscriptionConnector) SaveSubscriptions(owner string, subscriptions
 		}
 
 		dbSubscriptions = append(dbSubscriptions, dbSubscription)
+
+		if dbSubscription.ResourceType == event.ResourceTypeVersion {
```

Comment:
This check is too broad, because it's still reasonable to add a subscription to a single version with this ID even if the version is a parent patch. This should maybe only add extra child subscriptions if the selectors are for a patch outcome

Reply:
hmm, good point, like for runtime. I'll fix it.